Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim
+
+WORKDIR /service
+COPY . /service
+RUN pip install -r requirements.txt
+
+EXPOSE 5000
```

Comment:
Should it be really hard-coded here which port is exposed? Isn't this a deployment detail that one likes to specify in the helm config?
If it is not hurting we could of course have it as a documentation feature that states which port is used by default.

@walzph what do you think?

Reply:
Yes, I think `5000` was there just as an example. Wasn't meant to be a firm choice.

Reply:
Actually the `EXPOSE` directive is only there for informative reasons:

> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports.
https://docs.docker.com/engine/reference/builder/#expose

Sure, we can add a default port there, but it might as well make sense to make this configurable (e.g. via the external config.yaml, etc.)

Reply:
Great, thanks @walzph for the explanaition. I have removed the EXPOSE tag.